### DISCUSSION TITLE
Unexpected error while listening to radio

### DISCUSSION THREAD

**Discussion by stefanos82** [audacious-media-player/audacious#89] [General]
I listen to an online radio station and at random times, I get this pseudo-crashing

![image](https://github.com/audacious-media-player/audacious/assets/1777466/f7d91044-cc5e-4b27-8f02-c79ed22de4ac)

Basically it works, but tries to reconnect without success unless I stop it and re-run it.

Is it something that has to do with buffering maybe? My Network Settings Buffer Size is set to `1024 KiB`.

My version is `Audacious 4.3.1 (unknown build)`; I built it myself via repo...by the way, it does not help that we don't know the actual commit version this issue gets reproduced :disappointed:

**Comment by jlindgren90** [comment]
My solution to PulseAudio issues is to not use PulseAudio 🤷

**Reply by stefanos82** [reply]
...now why it didn't respect pipewire and I had to re-choose it? :open_mouth: 

I wonder what caused the fallback to pulseaudio!

**Comment by stefanos82** [comment]
Pipewire does not work as expected I'm afraid...Audacious volume does not sync with Volume Control app.

![image](https://github.com/audacious-media-player/audacious/assets/1777466/a57289b5-3c08-400a-9910-7cb683848171)

**Reply by mschwendt** [reply]
That works here with Fedora 39 Workstation (based on GNOME Shell).

**Reply by stefanos82** [reply]
Hmm, maybe it's something that has to do with my desktop environment? I use XFCE4 4.18 under GNU / Debian testing 64-bit.
